runtime/solo/src/lib.rs

```diff
@@ -343,54 +319,15 @@ impl pallet_treasury::Config for Runtime {
 	type OnSlash = ();
 	type ProposalBond = FivePercent;
 	type ProposalBondMinimum = MinimumProposalBond;
-	type ProposalBondMaximum = ();
-	type SpendPeriod = Weekly;
+	type ProposalBondMaximum = MaximumProposalBond;
+	type SpendPeriod = OneWeek;
```

Comment:
I think that weekly is a misnomer, as we are talking about a period, not a frequency.